enhancements/20250606-sidecar-support.md

```diff
@@ -0,0 +1,92 @@
+# Sidecar support
+
+Enable Kubvirt sidecar support to allow fine tuning of the Kubevirt generated domain definitions
+
+## Summary
+
+This enhancement aims to provide a generic way to tweak Kubevirt generated domain definitions to allow fine tuning of limited set of settings.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/5619
+
+## Motivation
+
+### Goals
+
+To provide a generic way to leverage Harvester built and 3rd party sidecars with Harvester
```

Comment:
upstream also supports to run shell/python from configmap, it is easier; do we plan to add it?
https://kubevirt.io/user-guide/user_workloads/hook-sidecar/#go-python-shell-pick-any-one